src/aap_eda/core/management/commands/create_initial_data.py

```diff
@@ -1054,6 +1056,37 @@
     },
 ]
 
+APPLICATION_SETTINGS_ARGS = [
```

Comment:
@hsong-rh We can add more settings that are needed for analytics. Let me know the name and type

Reply:
@bzwei we need 3 more settings:

INSIGHTS_CERT_PATH, string
AUTOMATION_ANALYTICS_LAST_GATHER: string (timestamp)
AUTOMATION_ANALYTICS_LAST_ENTRIES: dict